frontend/src/hooks/useFormData/index.tsx

```diff
@@ -28,7 +28,7 @@ interface Args<F, V> {
     initialFormData: ConstrainedFormData<F>;
     optionalFields: string[];
     missingFieldErrorMessage?: ErrorMessage;
-    submit: (validatedFormData: ConstrainedFormData<V>) => Promise<any>; // FIXME
+    submit: (validatedFormData: ConstrainedFormData<V>) => Promise<any>;
```

Comment:
I should add a type for server responses, but `any` is being inferred on `main` as well, so it's not worse than before (just more obvious now that there's a type annotation). Will make another little issue for this

Reply:
Remember we can change/restructure the server responses at any point if it makes it easier to work with on the frontend. We're in full control here.

Reply:
Yes and also given Servant plan on other branch, better to leave this alone for now! Covered in all respects